act/onto_index.sql

```diff
@@ -7,14 +7,10 @@ drop index SHRINE_ONT_ACT.ACT_COVID_fullname_IDX;
 drop index SHRINE_ONT_ACT.ACT_COVID_hlevel_IDX;
 drop index SHRINE_ONT_ACT.ACT_DEMO_APPLIED_IDX ;
```

Comment:
We should move all of these indexes to appropriate sql mapping to maintain the new format.

Reply:
yes... I guess I didn't quite finish that.